### DISCUSSION TITLE
Feature suggestion: Update dependencies of existing servers & save artifacts to .alpackages

### DISCUSSION THREAD

**Discussion by cegekaJG** [microsoft/AL-Go#1562] [Ideas]
I can't speak for cloud environments because I don't work with them often, but I'd like to be able to check for updates for all dependencies defined in the AL-GO settings and the `app.json` manifest, download any new ones and finally publish them to a given server, in my case a local docker container. `CreateDevEnv` already downloads and publishes all dependencies, but I don't want to build a new container from scratch - it's useful for testing, but during development, I want to use my existing container.
Is it possible to refactor the code within `CreateDevEnv` so it's possible to update the dependencies on demand?

On that note, when creating a new local container, wouldn't it be better to save the artifacts to `.alpackages` and not delete them afterwards? That way, the workspace already has the symbol correctly set up.

**Comment by freddydk** [comment]
Some good ideas, which we will consider when we do the next iteration on development environemnts...
Thanks
